labellab-flask/api/controllers/imagescontroller.py

```diff
@@ -0,0 +1,205 @@
+from datetime import datetime
+
+from flask.views import MethodView
+from flask import make_response, request, jsonify, current_app
+from flask_jwt_extended import (
+    jwt_required,
+    get_jwt_identity,
+    get_raw_jwt
+)
+from api.models.Image import Image
+from api.helpers.image import (
+    convert_and_save,
+    get_dimensions,
+    save as save_image,
+    find_all_by_project_id as find_images,
+    find_by_id,
+    remove_image,
+    delete_by_id
+)
+
+
+class SubmitImage(MethodView):
+    """This class saves a new image."""
+    @jwt_required
+    def post(self, project_id):
+        """
+        Handle POST request for this view.
+        Url --> /api/v1/image/create/<int:project_id>
+        """
+        # getting JSON data from request
+        post_data = request.get_json(silent=True,
+                                     force=True)
+        current_user = get_jwt_identity()
+        try:
+            images = post_data["images"]
+            image_names = post_data["imageNames"]
```

Comment:
Change ```imagesNames``` to snake case to match the other endpoints.

Reply:
sure !